README.md

```diff
@@ -37,6 +37,12 @@ git submodule add https://github.com/sobolevn/dotbot-brewfile.git
 3. Add required options to your [`install.conf.yaml`](/example.yaml):
 
 ```yaml
+- defaults:
```

Comment:
Can you please explain this `defaults` fields once again, please? 🙂

Reply:
Check, please